hashtagmap-instagram-crawler/src/main/java/com/songpapeople/hashtagmap/crawler/InstagramCrawler.java

```diff
@@ -0,0 +1,40 @@
+package com.songpapeople.hashtagmap.crawler;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.songpapeople.hashtagmap.dto.CrawlingDto;
+import com.songpapeople.hashtagmap.dto.PostDto;
+import com.songpapeople.hashtagmap.dto.PostDtos;
+import org.jsoup.nodes.Document;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InstagramCrawler {
+    private static final String INSTAGRAM_URL_FORMAT = "https://www.instagram.com/explore/tags/%s/?hl=ko";
+    public static final String POST_URL_KEY = "shortcode";
+    public static final String DISPLAY_URL_KEY = "display_url";
+    public static final String SOURCE_KEY = "edges";
+
+    public CrawlingDto createHashtagDto(String placeName) {
+        Document document = Crawler.crawling(String.format(INSTAGRAM_URL_FORMAT, placeName));
+        String body = document.body().toString();
+        String hashtagCount = RegexPattern.HASH_TAG_COUNT.extract(body);
+        String popularPostsInfo = RegexPattern.HASHTAG_POPULAR_POSTS_INFO.extract(body);
```

Comment:
`Crawler`로부터 `Document`를 만드는 방식보다 **특정 객체**를 생성해서
해당 객체에서 `getHashtagCount(RegexPattern regexPattern)`, `getPopularPostInfo(RegexPattern regexPattern)` 하는 방식으로 하면 좀 더 테스트하기 용이한 코드가 작성 될 수 있지 않을까요?
지금 코드는 내부 구현의 노출이 많이 되어 있는 것 같아요! 
조금 더 추상화하면 좋을 것 같습니다!

Reply:
```suggestion
        Foo foo = Crawler.crawling(String.format(INSTAGRAM_URL_FORMAT, placeName));
        String hashtagCount = foo.getHashtagCount();
        String popularPostsInfo = foo.getPopularPostInfo();
```

위와 같은 모습이 될 수 있을 것 같네요!

Reply:
아니면 더 나아가 `getPopularPostsInfo()` 가 `postDto` 를 반환해도 좋을 것 같다고 생각이 드네요!
제 생각에는 `postDto`는 DTO라고 보기 보다 도메인 영역에 해당하는 객체라는 생각이 듭니다!
`PopularPost` 라는 도메인 객체라고 보는건 어떻게 생각하시나요? 궁금하네요!